components/fxa-client/src/fxa_client.udl

```diff
@@ -0,0 +1,1075 @@
+
+
+
```

Comment:
Sadly, UniFFI is not capable of copying these comments over into the generated Kotlin or Swift files, because the parser we're using doesnt make them available. I expect that will change over time as we revisit the parsing story, so I've been happy to invest a little bit in the docstrings here, confident that they'll be or more use to us over time.